readme.md

```diff
@@ -139,7 +177,20 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+
+var Instructor = function(name){
+  this.name = name;
+}
+
+Instructor.prototype = {
+  recievePresent: function(gift){
```

Comment:
Awesome use of a prototype method, just be mindful here of the difference between `Instructor.prototype ={ ... }` and `Instructor.prototype.receivePresents = function( ){..}` The first is overwriting the prototype, the second adds a method to the existing prototype...